webapp-django/crashstats/crashstats/templates/crashstats/daily.html

```diff
@@ -2,6 +2,7 @@
 {% block site_css %}
   {{ super() }}
   {% compress css %}
+  <link rel="stylesheet" type="text/css" href="{{ static('crashstats/css/lib/metricsgraphics.css') }}">
```

Comment:
This was actually better along with jquery and others. It's the ``daily.less`` file that should be included after the more generic files. Can you please move that one down to be loaded last?